modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/config/BaseConfiguration.java

```diff
@@ -173,4 +180,6 @@ public PassThroughTransportMetricsCollector getMetrics() {
         return metrics;
     }
 
+    public Boolean isCorrelationLoggingEnabled(){return correlationLoggingEnabled;}
```

Comment:
Formatting issue shall we fix this?

Reply:
fixed in https://github.com/wso2/wso2-synapse/pull/1305